.env.example

```
@@ -1,47 +1,71 @@
-# File Upload & Security
+# Application URLs
+API_URL=http://localhost:3002
+FRONTEND_URL=http://localhost:3000
+ADMIN_URL=http://localhost:3001
+
+# Server Configuration
+PORT=3002
+NODE_ENV=development
+
+# Logging (Phase 5)
+LOG_LEVEL=info  # debug, info, warn, error
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)